README.md

```diff
@@ -1,105 +1,6 @@
 # vhost-user-backend
 
-## Design
+This crate is deprecated and will receive no bug fixes or updates.
```

Comment:
I would say instead that the source code for this crate is moved to: [vhost-user-backend](https://github.com/rust-vmm/vhost/tree/main/crates/vhost-user-backend). You might want to use the root of the repository though in case we change the hierarchy of crates so this can still be reachable.

Reply:
done